smtsolver.js

```diff
@@ -44,7 +52,9 @@ function solve (query) {
     if (
       !solverOutput.startsWith('sat') &&
       !solverOutput.startsWith('unsat') &&
-      !solverOutput.startsWith('unknown')
+      !solverOutput.startsWith('unknown') &&
+      !solverOutput.startsWith('(error') &&
```

Comment:
What returns `(error`?

Reply:
Eldarica returns error strings in that format

Reply:
Bruh.

Reply:
To make them look like smtlib sexpr I guess

Reply:
It's just the few starting characters so I suppose the `error` is followed by more info and is wrapped in a parenthesis together with it?

Reply:
Yep, like `(error could not generate model)` or something

Reply:
Might be a good idea to add an example in a comment next to it to make it less of a WTF for anyone who spots this later, but other than that, looks like the code is fine here.

Reply:
Sure